lib/transactions/utils.py

```diff
@@ -1,68 +0,0 @@
-from lib.transactions import constants
-from lib.transactions.models import Transaction
-
-from solitude.logger import getLogger
-
-log = getLogger('s.transactions')
-
-
-def completed(detail, item):
-    log.info('Completing transaction.')
-    try:
-        record = Transaction.objects.get(
-            uuid=detail.get('tracking_id'),
-            status__in=(
-                constants.STATUS_PENDING,
-                constants.STATUS_CHECKED
-            )
-        )
-    except Transaction.DoesNotExist:
-        return False
-
-    record.status = constants.STATUS_COMPLETED
-    record.save()
-    return True
```

Comment:
It doesn't, but the only place that calls it is the Paypal IPN code. Bango and Boku have their own event notice code implemented. Removing this didn't cause any errors in those code paths.